projects/the-application/src/assets/i18n/main/hi.json

```diff
@@ -1,17 +1,19 @@
 {
-  "ADD_TO_HOME_SCREEN": "ADD_TO_HOME_SCREEN",
-  "CLOSE": "CLOSE",
-  "DESCRIPTION": "DESCRIPTION",
-  "FULL_STOP": "FULL_STOP",
-  "GAME": "GAME",
-  "HIGH_SCORES": "HIGH_SCORES",
-  "INSTALL": "INSTALL",
-  "INSTALL_MEMORY_GAME": "INSTALL_MEMORY_GAME",
-  "PLEASE_RELOAD": "PLEASE_RELOAD",
-  "PLEASE_FINISH_OR_RESET_GAME": "PLEASE_FINISH_OR_RESET_GAME",
-  "RECENT_SCORES": "RECENT_SCORES",
-  "REFRESH": "REFRESH",
-  "SETTINGS": "SETTINGS",
-  "TITLE": "TITLE",
-  "UPDATE_AVAILABLE": "UPDATE_AVAILABLE"
+  "ADD_TO_HOME_SCREEN": "होम स्क्रीन में शामिल करें",
+  "CLOSE": "बंद करे",
+  "DESCRIPTION": "मनुष्यों, कुत्तों और बिल्लियों के लिए मेमोरी कार्ड गेम",
+  "FULL_STOP": "|",
+  "GAME": "खेल",
+  "HIGH_SCORES": "उच्च स्कोर",
+  "INSTALL": "इंस्टॉल",
+  "INSTALL_MEMORY_GAME": "मेमोरी गेम इंस्टॉल करें",
+  "PLEASE_RELOAD": "कृपया पुनः लोड करें",
+  "PLEASE_FINISH_OR_RESET_GAME": "कृपया गेम समाप्त करें या रीसेट करें",
+  "RECENT_SCORES": "हाल ही में स्कोर",
+  "REFRESH": "रीफ़्रेश करें",
+  "SETTINGS": "सेटिङ्",
+  "TAP": "स्पर्श",
+  "THEN": "बद में",
+  "TITLE": "Memory खेल",
```

Comment:
Is "Memory" supposed to be translated?

Reply:
Oh, yeah! You're right. I must've missed it.

Reply:
I think @haldarsumana left this because she didn't know and wanted the review for correction. Can you provide the correct translation in the comments?

Reply:
It should be replaced with मेमोरी